runtime/compiler/optimizer/TreeLowering.hpp

```diff
@@ -75,8 +75,9 @@ class TreeLowering : public TR::Optimization
     *
     * @param block is the block containing the TreeTop to be moved
     * @param tt is a pointer to the TreeTop to be moved
+    * @param isAddress shows if the node is address, otherwise it is assumed to be an integer
     */
-   void moveNodeToEndOfBlock(TR::Block* const block, TR::TreeTop* const tt, TR::Node* const node);
+   void moveNodeToEndOfBlock(TR::Block* const block, TR::TreeTop* const tt, TR::Node* const node, bool isAddress = false);
```

Comment:
Not sure whether this function is used often enough to justify a default argument value

Reply:
Since this method is first added for fast acmp path which only checks integer store op. That's why I thought to keep it as a default. I'll remove the default value for `isAddress` in next update.